plugins/modules/ufw.py

```diff
@@ -113,9 +113,10 @@
     aliases: [port]
   proto:
     description:
-      - TCP/IP protocol.
+      - TCP/IP protocol. 
```

Comment:
This got a trailing space:
```suggestion
      - TCP/IP protocol.
```